src/js/node/_http_server.ts

```diff
@@ -998,7 +1069,24 @@ const NodeHTTPServerSocket = class Socket extends Duplex {
     return this;
   }
 
-  _write(_chunk, _encoding, _callback) {}
+  _write(_chunk, _encoding, _callback) {
+    const handle = this[kHandle];
+    // only enable writting if we can drain
+    if (handle && handle.ondrain) {
+      try {
+        handle.write(_chunk, _encoding);
```

Comment:
we can simplify this code a little.

```suggestion
        let err;
        try {
          if (handle && handle.ondrain && !handle.write(_chunk, _encoding)) {
               this.#pendingCallback = _callback;
               return false;
          }
        } catch(e) { 
          err = e;
        }
        err ? _callback(err) : _callback();
```